runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -164,32 +157,6 @@ object GrpcClientSettings {
   private[grpc] def staticServiceDiscovery(host: String, port: Int) =
     new HardcodedServiceDiscovery(Resolved(host, immutable.Seq(ResolvedTarget(host, Some(port), None))))
 
-  /**
-   * INTERNAL API
-   *
-   * @param config The config to parse, assumes already at the right path.
-   */
-  @InternalApi
-  private def getSSLContext(config: Config)(implicit actorSystem: ActorSystem): SSLContext = {
-    val sslConfigSettings: SSLConfigSettings = SSLConfigFactory.parse(config)
-    val sslContext: SSLContext = new ConfigSSLContextBuilder(
-      new AkkaLoggerFactory(actorSystem),
-      sslConfigSettings,
-      new DefaultKeyManagerFactoryWrapper(sslConfigSettings.keyManagerConfig.algorithm),
-      new DefaultTrustManagerFactoryWrapper(sslConfigSettings.trustManagerConfig.algorithm)).build()
-    sslContext
-  }
```

Comment:
Maybe we can still use this code but, instead of returning `SSLContext` return a `trustManager` and a `keyManager` build using `ssl-config`'s `Default{Key,Trust}ManagerFactoryWrapper`.